components/automate-elasticsearch/habitat/plan.sh

```diff
@@ -15,8 +15,7 @@ pkg_build_deps=(
   core/patchelf
 )
 pkg_deps=(
-  core/coreutils-static
-  core/busybox-static
```

Comment:
Do we know why busybox was here at all? Just copy/pasta?

Reply:
i think i copy-pasta'd it last week